app/index.html

```diff
@@ -7,6 +7,8 @@
     </head>
     <body>
       <div id="root"></div>
+      <script src="https://code.jquery.com/jquery-3.1.1.slim.min.js"></script>
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js"></script>
```

Comment:
Let's not use online sources. Use either `npm install` or download and paste into the `app/resources` folder. We don't want any errors when developing offline.